src/Chapter1.hs

```diff
@@ -568,8 +569,8 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
-
+isVowel c = elem c ['a', 'e' .. 'z']
```

Comment:
This would include much more than needed for vowels. It is better to manually specify all the letters separately and check on that:

```haskell
isVowel c = c `elem` ['a', 'e', 'i', 'o', 'u']
```

One note in here, that sometimes, `elem` could be slower than the explicit pattern matching. I remember there were some benchmarks on one particular case, that showed how moving to pattern matching on each case separately drastically decrease time 🐎